spec/api-app-spec.js

```diff
@@ -87,6 +87,30 @@ describe('app module', function() {
     });
   });
 
+  describe('app.addDistributedNotificationObserver', function() {
+    it('emits custom events for notifications', function(done) {
+      if (process.platform != 'darwin')
+        done();
+
+      this.timeout(5000);
+
+      app.addDistributedNotificationObserver(
+        'ExampleNotification',
+        'example-notification'
+      );
+
+      app.on('example-notification', function(){
+        done();
+      });
+
+      //Emit an event
+      ChildProcess.exec('echo \'import Foundation;' +
+        'NSDistributedNotificationCenter.defaultCenter().postNotificationName'+
+        '("ExampleNotification", object: nil);\' | swift -'
```

Comment:
Hmm, looks like it was just a timeout error, I increased the timeout value to 12000 and was able to run script/cibuild successfully on my own machine, will update the PR. :)